### PR TITLE
Adds snow flakes! Happy Holidays

### PR DESCRIPTION
**Pull request by itsjoekent**
#### What's this PR do?

Adds snowflakes to the site!
#### Where should the reviewer start?

Snowflake.js !!
#### How should this be manually tested?

Do snowflakes epically spawn on your screen?
#### Screenshots (if appropriate)

screen shot 2015-12-11 at 5 17 09 pm

### CODE REVIEW

**Comment by itsjoekent** [comment]
NEVER MIND THERES A FATAL FLAW IN THE DESIGN. thanks @DFurnes 

edit: since diego asked, the flaw was that a canvas rendered the snowflakes, and blocked you from clicking anything on the page. as diego put it 

> so not a snowflake, but more of a blocking blizzard

**Comment by DFurnes** [comment]
I think this is also something we'd definitely want to have a feature flag to enable/disable.

**Comment by itsjoekent** [comment]
This also uses the ascii :snowflake: symbol now (and I added the feature flag per @DFurnes suggestion, its under "FUN" in dosomething settings)

screen shot 2015-12-14 at 10 35 04 am

**Comment by weerd** [comment]
Hokay :+1:

File: lib/themes/dosomething/paraneue_dosomething/js/utilities/Snowflake.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+function getRandomInt(min, max) {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+function buildSnowflake() {
+  var $snowflake = $('<p class="snowflake fadeOutDownBig">&#10052;</p>');
+  $snowflake.css("left", getRandomInt(0, $(document).width()));
+  $('body').append($snowflake);
+
+  // append paragraph to body with or emoji
```
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/js/utilities/Snowflake.js:13]: huh? `...with or emoji`?

**Pull request merged**